Subject: [PATCH] Remove C99/C++11 standard definition from the default lists.
---
Index: tools/cmake/build.cmake
<+>UTF-8
===================================================================
diff --git a/tools/cmake/build.cmake b/tools/cmake/build.cmake
--- a/tools/cmake/build.cmake	(revision ac5d805d0eda75442dd17cfea5baaca9748f8215)
+++ b/tools/cmake/build.cmake	(date 1691152276304)
@@ -107,9 +107,9 @@
                                     # go into the final binary so have no impact on size
                                     "-ggdb")
 
-    list(APPEND c_compile_options   "-std=gnu99")
+#    list(APPEND c_compile_options   "-std=gnu99")
 
-    list(APPEND cxx_compile_options "-std=gnu++11")
+#    list(APPEND cxx_compile_options "-std=gnu++11")
 
     list(APPEND link_options "-Wl,--gc-sections")
 
